Sources/SpeziLicense/Resources/Localizable.xcstrings

```diff
@@ -0,0 +1,64 @@
+{
+  "sourceLanguage" : "en",
+  "strings" : {
+    "CONTRIBUTIONS_LIST_DESCRIPTION" : {
```

Comment:
We have started to move to more human-readable localization strings and putting them in there a a whole instead of the upper-case abbreviations as long as they are not too long; it would be great if we start doing this here as well.